css/css-regions-1/contentEditable/contentEditable-001.html

```diff
@@ -7,7 +7,7 @@
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#the-flow-into-property">
 		<link rel="help" href="http://www.w3.org/TR/css3-regions/#flow-from">
 		<link rel="help" href="http://www.w3.org/TR/html5/editing.html#contenteditable">
-		<meta name="flags" content="dom interact">
```

Comment:
There's nothing async in there, so it should be fine just taking the screenshot after the load event fires.